.gitignore

```diff
@@ -24,6 +24,7 @@ __pycache__
 packagecache
 /MANIFEST
 /build
+!mesonbuild/build
```

Comment:
I don't think is actually necessary if my gitignore knowledge is right.

Reply:
It's an artifact of setuptools -- `python setup.py build --build-base=build/` is the default build base, it's how setuptools handles build directories.

I do not understand why we need to override an anchored path either, though.

Reply:
I don't know either, but if you remove this line and run `touch mesonbuild/build/foo.txt` git will not add it and makes you use --force